tests-integration/src/container.rs

```diff
@@ -23,7 +23,7 @@ pub(crate) fn test_bootc_upgrade() -> Result<()> {
         assert!(!st.success());
         let stderr = String::from_utf8(o.stderr)?;
         assert!(
-            stderr.contains("this command requires a booted host system"),
+            stderr.contains("This command requires a booted host system"),
```

Comment:
This change to this "this" broke the integration test; doesn't seem related.

Reply:
yeah whoops, removed the `t` arbitrarily to make sure I'm running what I'm editing, but then put it back the wrong way...